include/QtNodes/internal/NodeStyle.hpp

```diff
@@ -27,27 +27,27 @@ class NODE_EDITOR_PUBLIC NodeStyle : public Style
     QJsonObject toJson() const override;
 
 public:
-    QColor NormalBoundaryColor;
-    QColor SelectedBoundaryColor;
-    QColor GradientColor0;
-    QColor GradientColor1;
-    QColor GradientColor2;
-    QColor GradientColor3;
-    QColor ShadowColor;
-    QColor FontColor;
-    QColor FontColorFaded;
-
-    QColor ConnectionPointColor;
-    QColor FilledConnectionPointColor;
-
-    QColor WarningColor;
-    QColor ErrorColor;
-
-    float PenWidth;
-    float HoveredPenWidth;
-
-    float ConnectionPointDiameter;
-
-    float Opacity;
+    QColor  NormalBoundaryColor;
+    QColor  SelectedBoundaryColor;
+    QColor  GradientColor0;
+    QColor  GradientColor1;
+    QColor  GradientColor2;
+    QColor  GradientColor3;
+    QColor  ShadowColor;
+    QColor  FontColor;
+    QColor  FontColorFaded;
```

Comment:
Please revert changes like these. There is no need to change these lines of code.

As a word of advice - it is usually desirable to only change code that needs to be changed to address the PR. This avoids causing confusion/overload when others review the change.

Additionally, stylistic coding is really up to repo owner as of now. And it seems like a single space between type and variables is the overall style (and it typically is).